fms_extras/models/speculator.py

```diff
@@ -264,6 +264,13 @@ def flatten_batch(inp: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor, torch.
 
 _llama_13b = {"emb_dim": 5120, "vocab_size": 32000, "n_predict": 3, "inner_dim": 4096}
 
+_gpt_bigcode_20b = {
```

Comment:
can we add a comment on what this config comes from (e.g. where a user might find appropriate weights for it or how we arrived at the set of configs, a paper reference if relevant, etc)